properties/basic.py

```diff
@@ -623,6 +669,26 @@ def error(self, instance, value, error=None, extra=''):
             )
         )
 
+    def deserialize(self, value):
+        """De-serialize the property value from JSON
+
+        If no deserializer has been registered, this converts the value
+        to the wrapper class with given dtype.
+        """
+        if self.deserializer:
+            return self.deserializer(value)
+        if value is None:
+            return None
+        return self.wrapper(value).astype(self.dtype[0])
+
+    @staticmethod
+    def to_json(value):
+        return value.tolist()
+
+    @staticmethod
+    def from_json(value):
```

Comment:
But the `deserialize` method for this class doesn't fall back to `from_json`, it uses `self.wrapper`, so when would `from_json` be called?